src/Illuminate/Container/Container.php

```diff
@@ -1280,6 +1290,16 @@ protected function fireCallbackArray($object, array $callbacks)
         }
     }
 
+    /**
+     * @return bool
+     */
+    public function hasConfigRepository(): bool
```

Comment:
Does this method need to be public? 
Does it need to be a method at all? 

Adding a new method (especially a public method) adds to the long term maintenance burden, it will be hard to rename/move it later because of a potential BC-break.

Reply:
Agreed. I created this because I thought it improved readability a bit. I will merge it back into the main functionality.